crypto/bindings/conversion.ts

```diff
@@ -526,6 +653,80 @@ function createRustConversion(wasm: wasm) {
       return mlRo;
     }
 
+    function commitmentsToRust(
+      commitments: ProverCommitments
```

Comment:
IMO, it would be good to prefix with `Ml` and `Wasm` all the types to ease the reading of the code.